auth/src/main/java/com/firebase/ui/auth/ui/email/CheckEmailFragment.java

```diff
@@ -94,6 +96,52 @@ public static CheckEmailFragment getInstance(@NonNull FlowParameters flowParamet
         return fragment;
     }
 
+    public static void checkAccountExists(Activity activity,
+                                          final BaseHelper helper,
+                                          final CheckEmailListener listener,
+                                          @NonNull final String email,
+                                          @Nullable final String name,
+                                          @Nullable final Uri photoUri) {
+        helper.showLoadingDialog(R.string.progress_dialog_checking_accounts);
+
+        if (!TextUtils.isEmpty(email)) {
```

Comment:
nit: generally if the whole method is wrapped in an `if()` it's better to just short-circuit to reduce nesting.  So

```java
if (TextUtils.isEmpty(email)) {
   return;
}

helper.getFirebaseAuth()
  // ...

}